### PR TITLE
Display episode thumbnails on home screen, add option for series

### PR DESCRIPTION
**Pull request by linetrimmer**
**Changes**

- Adds an option to go use series or episode thumbnails when displaying episodes in home screen rows

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt [hunk 1]
```diff
@@ -179,6 +179,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 * Set which ratings provider should show on MyImageCardViews
 		 */
 		var defaultRatingType = Preference.enum("pref_rating_type", RatingType.RATING_TOMATOES)
+
+		/**
+		 * Enable series thumbnails in home screen rows
+		 */
+		var seriesThumbnailsEnabled = Preference.boolean("pref_enable_series_thumbnails", false)
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt:186]: Discussed on Matrix, should default to true

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -461,4 +461,6 @@
     <string name="login_help_title">Need help?</string>
     <string name="login_help_description">Jellyfin requires a server to connect with. Visit our documentation at docs.jellyfin.org to get started with Jellyfin.</string>
     <string name="connect_title">Connect</string>
+    <string name="lbl_use_series_thumbnails">Prefer series thumbnails on home screen</string>
+    <string name="lbl_use_series_thumbnails_description">Use series thumbnails for episodes in home screen rows</string>
```
Comment (nielsvanvelzen) [inline app/src/main/res/values/strings.xml:465]:
```suggestion
    <string name="lbl_use_series_thumbnails">Prefer series thumbnails</string>
    <string name="lbl_use_series_thumbnails_description">Use series thumbnails when displaying episodes</string>
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
I'm not really sure if `seriesThumbnailsEnabled` is descriptive of the behavior but I couldn't think of any better name.

**Review by nielsvanvelzen — APPROVED** [review]

**Review by thornbill — APPROVED** [review]
:+1: lgtm once the conflicts are resolved!

**Review by thornbill — APPROVED** [review]

**Pull request merged**